docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
     nginx:
         build: nginx
         ports:
-            - 80:80
+            - 8080:80
```

Comment:
ak menime nginx zmenme aj `elk` aj `phpmyadmin` napr na `8081`, `8082`

Reply:
To je rozumny napad. Ale to mozno uprav cez ine issue?